### DISCUSSION TITLE
More data in telemetry page?

### DISCUSSION THREAD

**Discussion by Armo00** [yagiziskirik/Kerbal-Telemetry#16] [Ideas]
Is it possible to add more data to the telemetry page, or maybe an advance page.
Things like Time to AP, PE, SOI; Max thrust, current thrust, max acceleration, current acceleration, G force and etc. 
A further thought is to let user chose the flight data they want to create a Flight Data page, just like how Mechjeb allows you to add new panels and config them. 
My MJ Flight Data Panel is like this:
![image](https://user-images.githubusercontent.com/17452430/174451716-20dd7ee4-9982-44c8-8ef5-4a8d18288c43.png)
And I would really like it to be displayed on another web page.

**Comment by yagiziskirik** [comment]
Hi @Armo00,

It is currently not possible but I am considering proper customisation on the webpage to see the relevant information that the user selected. The data you see on the Kerbal Telemetry screen are the ones that I use and could extract from the KSP. But more customisation is on the way!
